autorun.js

```diff
@@ -0,0 +1,37 @@
+import { Tracker } from 'meteor/tracker';
+import { autorun } from 'mobx';
+
+export default (trackerMobxAutorun) => {
+  let mobxDisposer = null;
+  let computation = null;
+  let hasBeenStarted;
+  let subscriptionHandle;
+  return {
+    start() {
+      let isFirstRun = true;
+      computation = Tracker.autorun(() => {
+        if (mobxDisposer) {
+          mobxDisposer();
+          isFirstRun = true;
+        }
+        mobxDisposer = autorun(() => {
+          if (isFirstRun) {
+            subscriptionHandle = trackerMobxAutorun();
+          } else {
+            computation.invalidate();
+            subscriptionHandle && subscriptionHandle.stop();
```

Comment:
Oh, you are right. We don't even need to stop subscription in scope of autorun.stop(), ok ;)